composer.json

```diff
@@ -19,7 +19,8 @@
     "require": {
         "php": ">=5.3.0",
         "psr/log": "^1.0",
-        "symfony/var-dumper": "^2.6|^3.0"
+        "symfony/var-dumper": "^2.6|^3.0",
+        "symfony/debug": "2.8.*|3.0.*"
```

Comment:
Yes please be as permissive as possible. If 2.6 is possible, use that as minimum. Also future symfony versions for 3.x should be compatible.